egs2/TEMPLATE/svs1/pyscripts/utils/prep_segments.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
We can discuss with others for the segmentation later. But this still can be a default one. Recommend to add some comments in the file to make it clear for usage